plasma/child_chain/child_chain.py

```diff
@@ -20,12 +22,22 @@ def __init__(self, authority, root_chain):
         self.current_block = Block()
         self.pending_transactions = []
 
-        # Register for deposit event listener
-        deposit_filter = self.root_chain.on('Deposit')
-        deposit_filter.watch(self.apply_deposit)
+        self.event_listener = RootEventListener(root_chain, finality=1)
```

Comment:
How do you feel about changing `finality` to `confirmations` as it's more descriptive?

Reply:
Agreed.